BlueprintUI/Tests/FocusStateTests.swift

```diff
@@ -72,4 +72,16 @@ final class FocusStateTests: XCTestCase {
         binding.onBlur()
         XCTAssertEqual(state.isFocused, false)
     }
+
+    func test_memoryRelease() {
```

Comment:
Not sure if it's worth it, but I always copy around this function, which waits for the release/dealloc to happen over a couple of runloop cycles; given the autorelease pool timing seem to change between iOS releases (or based on our code, depending on when values get set), which makes the tests more stable: https://github.com/squareup/market/blob/e7f264ca2ab8a7d1bcade4a7b5fe179e2f41302a/ios/MarketUITesting/Sources/XCTestCase%2BTestHelpers.swift#L68

Reply:
There should be no need for any autorelease pool here! AFAIK this test is deterministic and guaranteed by ARC semantics.